pipelines/crawler/sfb_sicar/tasks.py

```
@@ -1,75 +1,138 @@
+    """
+    # Run the clean in a child process launched with MALLOC_ARENA_MAX in its
+    # environment. glibc reads that only before its first malloc, so it must be
+    # present at process start — capping arenas here is what keeps the dense
+    # Amazonas app clean from ballooning RSS and OOMing the worker. See
+    # ``_clean_runner`` for the mechanism.
+    env = {
+        **os.environ,
+        "MALLOC_ARENA_MAX": "2",
+        "MALLOC_TRIM_THRESHOLD_": "131072",
+    }
+    proc = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)